.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+version: 2
+jobs:
+  build:
+    working_directory: /go/src/github.com/{{ORG_NAME}}/{{REPO_NAME}}
+    docker:
+      - image: circleci/golang:1.9
+    steps:
+      - checkout
+      - run: mkdir -p ./dist
+      - run:
+          name: Lint
+          command: make local-check
+      - run:
+          name: Build x86_64/linux
+          environment:
+            GOOS: linux
+            GOARCH: x86_64
+          command: go build -i -o dist/linuxkit-$GOOS-$GOARCH github.com/linuxkit/linuxkit/src/cmd/linuxkit
```

Comment:
any way to add the commits and version to this sanely?

Reply:
I'll investigate. Might be that we can use `make local-build` if Circle doesn't have the issue described in the travis.yml comment (which I should replicate here if it does)

Reply:
Turns out that not only does CircleCI not suffer from that issue, but the workaround (using `go build -i`) is broken, while not having the workaround works...